intl/msg_hash_us.h

```diff
@@ -13668,6 +13668,50 @@ MSG_HASH(
    MSG_CHEEVOS_HARDCORE_MODE_ENABLE,
    "Achievements Hardcore Mode Enabled, save state & rewind were disabled."
    )
+MSG_HASH(
+   MSG_CHEEVOS_HARDCORE_MODE_PAUSED,
+   "Hardcore paused. You cannot earn hardcore achievements for %s using %s"
+   )
+MSG_HASH(
+   MSG_CHEEVOS_LOGGED_IN_AS_USER,
+   "RetroAchievements: Logged in as \"%s\"."
+   )
+MSG_HASH(
+   MSG_CHEEVOS_COULD_NOT_ACTIVATE_ACHIEVEMENT,
+   "Could not activate achievement %u \"%s\": %s"
+   )
+MSG_HASH(
+   MSG_CHEEVOS_COULD_NOT_ACTIVATE_LEADERBOARD,
+   "Could not activate leaderboard %u \"%s\": %s"
+   )
+MSG_HASH(
+   MSG_CHEEVOS_GAME_HAS_NO_ACHIEVEMENTS,
+   "This game has no achievements."
+   )
+MSG_HASH(
+    MSG_CHEEVOS_ALL_ACHIEVEMENTS_ACTIVATED,
+    "All %d achievements activated for this session."
+)
+MSG_HASH(
+    MSG_CHEEVOS_NUMBER_ACHIEVEMENTS_UNLOCKED,
+    "You have %d of %d achievements unlocked."
+)
+MSG_HASH(
+    MSG_CHEEVOS_ALL_ACHIEVEMENTS_ACTIVATED_UNSUPPORTED,
+    "All %d achievements activated for this session (%d unsupported)."
+)
+MSG_HASH(
+    MSG_CHEEVOS_NUMBER_ACHIEVEMENTS_UNLOCKED_UNSUPPORTED,
+    "You have %d of %d achievements unlocked (%d unsupported)."
```

Comment:
Recommend just having a single string for "%d unsupported", then use the above messages to generate the output.

Untested code:
```
char unsupported[64];
snprintf(unsupported, sizeof(unsupported),
   msg_hash_to_str(MSG_CHEEVOS_UNSUPPORTED_COUNT), number_of_unsupported);
snprintf(msg, sizeof(msg), msg_hash_to_str(MSG_CHEEVOS_ALL_ACHIEVEMENTS_ACTIVATED), number_of_core);
snprintf(msg, sizeof(msg), "%s (%s)", msg, unsupported);
```

Reply:
For this one, I made the following additions on top of the code that you suggested:
- I have added a new buffer for the initial main message, as we get a compiler warning otherwise for writing to the same buffer in a circular way;
- I am checking the return value of the last snprintf and returning early if it's -1, as there could be a hypothetical scenario where the result is greater than the msg buffer (this was also a compiler warning)